OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs

```diff
@@ -95,6 +95,9 @@ public bool SimilarTerrain(CPos xy, CPos sourceLocation)
 			var sourceTiles = Self.World.Map.FindTilesInCircle(xy, range);
 			var destTiles = Self.World.Map.FindTilesInCircle(sourceLocation, range);
 
+			if (sourceTiles.Count() == 0 || destTiles.Count() == 0)
```

Comment:
Nit: Use `.Any()` instead of checking `.Count()` for code consistency and a minor performance improvement.